docs/developers/contributing/documentation/index.md

```diff
@@ -350,48 +357,60 @@ This will prevent all but the first napari window from being shown during the do
 
-**To preview on your browser:**
+* preview on your browser via [CircleCI](https://circleci.com/) in just one click -
+  this is the easiest method but in rare cases it may not match the documentation that
+  is actually deployed to [napari.org](https://napari.org).
+* download the built documentation artifact and view it locally - this is more
+  complicated, but the built docs will always match what is deployed to
+  [napari.org](https://napari.org).
 
-Click on **Details** next to the `Check the rendered docs here!` check:
+When you submit a pull request to the [napari/napari](https://github.com/napari/napari)
+repository, its continuous integration will only build the docs in CircleCI. Thus
+you will only be able to preview the documentation on CircleCI.
 
-![CircleCI check is highlighted](images/circleci-link.png)
+#### Preview on CircleCI
 
-This will open a preview of the website on your browser, and you can then
-navigate to the page where you expect to see your changes.
+Simply click on **Details** next to the `Check the rendered docs here!` at the bottom
+of your pull request:
 
-**To download the built documentation pages:**
+![CircleCI check is highlighted](images/circleci-link.png)
+
+#### Download documentation artifact
 
-1. Click on **Details** next to `Build PR Docs / Build & Upload Artifact (pull_request)`:
+1. Click on **Details** next to
+   `Build & Deploy PR Docs / Build & Upload Artifact (pull_request)`:
 
-![The "Build PR Docs / Build & Upload Artifact" check is highlighted](images/doc-ci-1.png)
+![The "Build & Deploy PR Docs / Build & Upload Artifact" check is highlighted](images/doc-ci-1.png)
 
 2. Click on **Summary** on the top left corner:
 
-![Summary link in the "Build PR Docs / Build & Uplod Artifact" GitHub Action page](images/doc-ci-2.png)
+![Summary link in the "Build & Deploy PR Docs / Build & Uplod Artifact" GitHub Action page](images/doc-ci-2.png)
 
-3. Scroll down to **Artifacts** and click on **docs** to download the built documentation:
+3. Scroll down to **Artifacts** and click on **html** to download the built documentation:
 
-!["docs" link in the Artifacts section of the "Build PR Docs / Build & Uplod Artifact" GitHub Action page is highlighted](images/doc-ci-3.png)
+!["html" link in the Artifacts section of the "Build & Deploy PR Docs / Build & Uplod Artifact" GitHub Action page is highlighted](images/doc-ci-3.png)
 
-4. Extract the compressed archive and open the `docs/index.html` file on your preferred browser.
+4. Extract the compressed archive and open the `html/index.html` file on your preferred browser.
    You can also use Python's `http.server` module to open a local server on
    [http://localhost:8000](http://localhost:8000):
 
 ```shell
-$ cd ~/Downloads/docs  # or the path where you extracted the 'docs' artifact
+$ cd ~/Downloads/html  # `cd` to the path where you extracted the 'html' artifact
 $ python3 -m http.server
 ```
 
+(docs_submit_pull_request)=
+
```

Comment:
```suggestion
```
I think this blank line is spurious?

Reply:
I don't think it matters for the output, but it's unnecessary indeed.

Reply:
Done!